Adding environment variable location instructions (#6390)
Particular/docs.particular.net: nservicebus/licensing/index_license-management_core_[8,].partial.md

NOTE: If the license is stored in `HKEY_CURRENT_USER`, NServiceBus processes must run as the user account used to add the license file to the registry in order to access the license.

It is safe to ignore any warnings regarding empty strings.

> ### Environment variable location

Reviewer: ```suggestion
### Environment variable
```

> Finally a license can also be specified via setting the `PARTICULARSOFTWARE_LICENSE` environment variable with the license text

Reviewer: ```suggestion
The license can also be specified by setting the `PARTICULARSOFTWARE_LICENSE` environment variable containing the license text
```

* Windows
  ```powershell
  $env:PARTICULARSOFTWARE_LICENSE = @"the license text goes here
  and is a multiline string"@
  ```
* Linux/macOS
  ```bash

>   export PARTICULARSOFTWARE_LICENSE=`cat ./license.xml`

Reviewer: Should we for consistency with the windows part only show the approach below?

Author: well, the reason to show this one is because I don't know the syntax in PS to do the same

Author: If someone else know how to do that in PS, we can add that later on
